firstvoices/backend/models/widget.py

```diff
@@ -0,0 +1,149 @@
+import rules
+from django.db import models
+from django.utils.translation import gettext as _
+
+from backend.models.base import (
+    BaseControlledSiteContentModel,
+    BaseModel,
+    BaseSiteContentModel,
+)
+from backend.permissions import predicates
+
+
+class WidgetTypes(models.IntegerChoices):
+    # enum intentionally has gaps to allow future changes to keep sequential order
```

Comment:
Based on Guy's comment that this does not need to be constrained on the backend, I think the widget type could just be a string field. No need for an enum. (Maybe we'll add it someday if the backend starts to have structured widget data.)